src/labkey/Exp.ts

```diff
@@ -0,0 +1,1006 @@
+     *             ]
+     *         },{
+     *             "name": "Sheet3",
+     *             "data": []
+     *         }
+     *     ]
+     * }
+     * ```
+     *
+     * An example of the same file in the 'jsonTSVExtended' format:
+     *
+     * ```js
+     * {
+     *     "filename": "SimpleExcelFile.xls",
+     *     "sheets": [
+     *         {
+     *             "name": "Sheet1",
+     *             "data": [
+     *                 {
+     *                     "value": "StringColumn",
+     *                     "formattedValue": "StringColumn"
+     *                 },{
+     *                     "value": "DateColumn",
+     *                     "formattedValue": "DateColumn"
+     *                 }
+     *             ],[
+     *                 {
+     *                     "value": "Hello",
+     *                     "formattedValue": "Hello"
+     *                 },{
+     *                     "formatString": "MMMM d, yyyy",
+     *                     "value": "16 May 2009 17:00:00",
+     *                     "timeOnly": false,
+     *                     "formattedValue": "May 17, 2009"
+     *                 }
+     *             ],[
+     *                 {
+     *                     "value": "world",
+     *                     "formattedValue": "world"
+     *                 },{
+     *                     "formatString": "M/d/yy h:mm a",
+     *                     "value": "21 Dec 2008 19:31:00",
+     *                     "timeOnly": false,
+     *                     "formattedValue": "12/21/08 7:31 PM"
+     *                 }
+     *             ]
+     *         },{
+     *             "name": "Sheet2",
+     *             "data": [
+     *                 [{
+     *                     "value": "NumberColumn",
+     *                     "formattedValue": "NumberColumn"
+     *                 }],[{
+     *                     "formatString": "$#,##0.00",
+     *                     "value": 55.44,
+     *                     "formattedValue": "$55.44"
+     *                 }],[{
+     *                     "value": 100.34,
+     *                     "formattedValue": "100.34"
+     *                 }],[{
+     *                     "value": -1,
+     *                     "formattedValue": "-1"
+     *                 }]
+     *             ]
+     *         },{
+     *             "name": "Sheet3",
+     *             "data": []
+     *         }
+     *     ]
+     * }
+     * ```
+     */
+    getContent(options: IGetContentOptions): void {
+        // NK: I'm choosing to not implement this call to "alert". There are plenty of places where we
+        // "require" arguments and fail less gracefully. In this case, not supplying a success just means the call
+        // is useless but I think a user of this API will quickly come to realize that they need to get access
+        // to the object somehow and investigate further.
+
+        // if (getOnSuccess(options)) {
```

Comment:
Yes, let's just remove the check for the success callback everywhere.  It's silly.

Reply:
👍